main_test.go

```diff
@@ -1,4 +1,4 @@
-// +build small
+// +build linux,medium
```

Comment:
I added macos to the build matrix and use runtime.GOOS to determine which plugin to load. So now this is just a medium test.